fdbclient/NativeAPI.actor.cpp

```diff
@@ -3189,6 +3189,32 @@ Future< StorageMetrics > Transaction::getStorageMetrics( KeyRange const& keys, i
 	return ::waitStorageMetrics( cx, keys, StorageMetrics(), m, StorageMetrics(), shardLimit );
 }
 
+ACTOR Future< Standalone<RangeResultRef> > waitStorageMetricsList(
```

Comment:
This function should also be renamed to something like `waitDataDistributionMetricsList`